### PR TITLE
Separate cabal package for chairman tests

### PR DESCRIPTION
**Pull request by newhoggy**
The failure in this PR shows that the chairman test is running in CI: https://github.com/input-output-hk/cardano-node/pull/1706

### CODE REVIEW

File: cardano-node-chairman/LICENSE [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "[]"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2019 Input Output (Hong Kong) Ltd.
```
Comment (Jimbo4350) [inline cardano-node-chairman/LICENSE:190]: This should be 2020

File: cardano-node-chairman/NOTICE [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Copyright 2019 Input Output (Hong Kong) Ltd.
```
Comment (Jimbo4350) [inline cardano-node-chairman/NOTICE:1]: This should be 2020

Comment (newhoggy) [reply cardano-node-chairman/NOTICE]: Fixed.  Thanks!

File: cardano-node-chairman/LICENSE (hunk 1 above)
Comment (newhoggy) [reply cardano-node-chairman/LICENSE]: Fixed.  Thanks!

**Comment by newhoggy** [comment]
bors merge

**Pull request merged**